src/types/aws/ap/index.ts

```diff
@@ -88,7 +89,7 @@ export type EndowmentOption = Pick<Endowment, "id" | "name">;
 
 //most are optional except id, but typed as required to force setting of default values - "", [], etc ..
 export type EndowmentProfileUpdate = Except<
-  Ensure<Endowment, "hide_bg_tip">,
+  Ensure<Endowment, "hide_bg_tip" | "slug" | "card_img">,
```

Comment:
```suggestion
  Ensure<Endowment, "hide_bg_tip" | "slug">,
```
unrelated to problem: `card_img` attribute is optional attribute of `type Endowment` which is not enforced in `type EndowmentProfileUpdate` - so not initialized and creates false diff (makes edit profile submit even nothing is changed - technically something changed under the hood `initial.card_image` is `undefined` while `fv.card_image` is initialized to `""`, change: `undefined` -> `""`)

A more sustainable definition would be `Required<Endowment>` so not to missing additing in `Ensure` when new optional attribute is added